client/src/core/format.js

```diff
@@ -179,6 +179,7 @@ const types_to_format = {
     return`${year_int}-${ window.lang === "en" ? year_int-2000+1 : year_int+1}`;
   },
   "fiscal_year_to_year": (fiscal_year) => _.chain(fiscal_year).split("-").head().value(),
+  "year_plural_text": (year_num) => year_num > 1 ? "years" : "year",
```

Comment:
This is something that would make more sense as a handlebar helper in `src/handlebars/helpers.side-effects.js`.

Defining it as a format is just indirectly making a handlebar helper out of it anyway, since all of the formats are registered as handlebars helpers for use in `text_maker` text. It isn't really a format though.

Reply:
It also doesn't need to be a format/handlebar helper unless its going to be used in multiple places. If we're dynamically pluralizing "year" a lot (we probably are), then if you add a utility for that then you should also try to update all the existing one off cases to use the utility.